internal/cmap/cmap.go

```diff
@@ -366,6 +366,11 @@ func (cmap *CMap) Type() int {
 	return cmap.ctype
 }
 
+// Nbits returns 8 bits for simple font CMaps and 16 bits for CID font CMaps.
+func (cmap *CMap) NBits() int {
```

Comment:
Technically one cmap could have a combination of both? i.e. single, and multi byte?

Reply:
The CMap contains can contain multiple codespaces, which can cover single or multi-byte ranges. So technically, yes. nbits seems to represent the byte size of the largest codespace.